Content.Server/GameTicking/GameTicker.cs

```diff
@@ -166,7 +169,7 @@ public void Update(FrameEventArgs frameEventArgs)
 
             if (RunLevel != GameRunLevel.PreRoundLobby ||
                 Paused ||
-                _roundStartTimeUtc > DateTime.UtcNow ||
+                _roundStartTimeTicks > IoCManager.Resolve<IGameTiming>().RealTime.Ticks ||
```

Comment:
First of all, you already have a dependency for `IGameTiming`, so use that instead of resolving here every tick...
Second of all, you should just compare `_roundInitializedTimeUtc` to `_gameTiming.RealTime` instead of using these "ticks".